runtime/oti/j9nonbuilder.h

```diff
@@ -4572,6 +4572,7 @@ typedef struct J9InternalVMFunctions {
 	UDATA  ( *addHiddenInstanceField)(struct J9JavaVM *vm, const char *className, const char *fieldName, const char *fieldSignature, UDATA *offsetReturn) ;
 #ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
 	struct J9ROMFieldOffsetWalkResult*  ( *fieldOffsetsStartDo)(struct J9JavaVM *vm, struct J9ROMClass *romClass, struct J9Class *superClazz, struct J9ROMFieldOffsetWalkState *state, U_32 flags, J9FlattenedClassCache *flattenedClassCache) ;
+	UDATA  ( *getArrayHeaderSize)(struct J9JavaVM *vm) ;
```

Comment:
the parameter should be a `J9IndexableObject *`

Reply:
I'm not able to convert 'J9IndexableObjectContiguous*' to 'J9IndexableObject*'

Reply:
Actually, do we even need any parameters? The only one variable used by ```((MM_GCExtensionsBase *)vm->gcExtensions)->indexableObjectModel.getHeaderSize()``` is `vm` and that is declared internally in `mminit.cpp`